Launcher/Classes/ViewControllers/DeviceSettingsViewController.swift

```diff
@@ -0,0 +1,26 @@
+import AppKit
+
+class DeviceSettingsViewController: NSViewController {
+    let applicationStateHandler = ApplicationStateHandler()
```

Comment:
We're using this pretty frequently. I am almost thinking it would make sense to add a singleton here?!